### PR TITLE
Closures

### PR DESCRIPTION
**Pull request by sravuri-nisum-com**

### CODE REVIEW

**Review by vllmstl — APPROVED** [review]
Are closures good or bad?
- Good example
- Bad example

File: JS/Closures/example1.html [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-<p>A function  displayName() can access variables defined inside the function me() :</p>
+<p>A function  displayName() can access variables defined inside the function pam() :</p>
 
-<!-- <button type="button" onclick="me()">Click Me!</button> -->
+<!-- <button type="button" onclick="pam()">Click Me!</button> -->
 <!-- <p id="demo2"></p> -->
 <p id="demo"></p>
 
 
 <script>
```
Comment (vllmstl) [inline JS/Closures/example1.html:12]: Always create a new file.
Move the below code to a new file and link it on this page.

File: JS/Closures/example2.html [hunk 2]
```diff
@@ -21,16 +21,18 @@
...
+  functionB();  // prints A, B, G
     // functionB closure does not get discarded
   A = 42;
-  functionB();  // prints 42, B, S
+  functionB();  // prints 42, B, G
 }
 functionA();
+
+
 </script>
```
Comment (vllmstl) [inline JS/Closures/example2.html:35]: Always create a new file.
Move the above code to a new file and link it on this page.

File: JS/Prototypes/PrototypeAttribute.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+<p>
+        Every JavaScript object has a prototype. The prototype is also an object.
+        
+        All JavaScript objects inherit their properties and methods from their prototype.
+        Prototype Attribute of Objects Created with new Object () or Object Literal</p>
+
+
+
+<script>
```
Comment (vllmstl) [inline JS/Prototypes/PrototypeAttribute.html:13]: Always create a new file.
Move the below code to a new file and link it on this page.

File: JS/Prototypes/index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>Objects </p>
+
+
+
+<script>
```
Comment (vllmstl) [inline JS/Prototypes/index.html:9]: Always create a new file.
Move the below code to a new file and link it on this page.

File: JS/Prototypes/inheritanceWithPrototype.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<body>
+       
+<p>Example of inheritance with the prototype property </p>
+<p>This prototype property is not enumerable; that is, it isn’t accessible in a for/in loop</p>
+
+<script>
```
Comment (vllmstl) [inline JS/Prototypes/inheritanceWithPrototype.html:8]: Always create a new file.
Move the below code to a new file and link it on this page.

File: JS/Prototypes/prototype.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>New Object with prototype </p>
+
+
+
+<script>
```
Comment (vllmstl) [inline JS/Prototypes/prototype.html:9]: Always create a new file.
Move the below code to a new file and link it on this page.

File: JS/closures.html [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+<body>
+
+<p>A function  displayName() can access variables defined inside the function pam() :</p>
+
+<!-- <button type="button" onclick="pam()">Click Me!</button> -->
+<!-- <p id="demo2"></p> -->
+<p id="demo"></p>
+
+
+<script>
```
Comment (vllmstl) [inline JS/closures.html:12]: Always create a new file.
Move the below code to a new file and link it on this page.

**Review by vllmstl — COMMENTED** [review]
+ ES6 syntax
- Why not use the ES6 syntax every where

File: JS/closures.html (hunk 7 above)
Comment (sravuri-nisum-com) [reply JS/closures.html:12]: Sure i will

File: JS/Closures/example2.html (hunk 2 above)
Comment (sravuri-nisum-com) [reply JS/Closures/example2.html:35]: sure i will

**Pull request merged**